drupal-vm/config.yml.dist

```diff
@@ -0,0 +1,314 @@
+---
```

Comment:
Technically, you would only need to include a few variables that need overriding (e.g. `vagrant_synced_folders`), instead of the entire config.yml file.

This way maintenance is a _lot_ easier over time, since you would just use Drupal VM's defaults (which should always work with the latest and n-1 releases of Drupal core).